sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/keyvault_secret.hpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Keyvault Secret definition
+ */
+
+#pragma once
+#include "keyvault_secret_properties.hpp"
```

Comment:
even if it is on the same path, we use the full path for public headers like `include "azure/keyvault/secrets/blabla.hpp"
Basically, we use CMake to make the inc folder part of the include directory and we take the headers from there  (only provate headers are the exception that we take by path)

Reply:
done